client/sources/common/interpreter.py

```diff
@@ -68,15 +68,17 @@ def unlock(self, interact):
                     phase.
         """
         print(self.setup.strip())
+        previous_line = None
         try:
             for line in self.lines:
                 if isinstance(line, str) and line:
                     print(line)
+                    previous_line = line
```

Comment:
Aren't some coding questions multiple lines? Wouldn't this then only capture the last line in the question?